### PR TITLE
Add reference to serverless-migrate-plugin

### PR DESCRIPTION
**Pull request by EliuX**
Greetings members of the serverless community. I got here a new tested (manually not with automated tests yet) plugin that allows to run migrations in serverless projects.
You will find in [serverless-migrate-plugin/example](https://github.com/EliuX/serverless-migrate-plugin/tree/master/example) a very clear explanation and sample code for you to try it out.

Greetings, 

EliuX

### CODE REVIEW

**Comment by eahefnawy** [comment]
Thanks @EliuX ... could you fix the conflicts? 😊

**Comment by EliuX** [comment]
> Thanks @EliuX ... could you fix the conflicts? 😊

Sure 👍

**Comment by eahefnawy** [comment]
@EliuX what about that added `serverless-middleware` plugin? I see it in the JSON file but not the README. 🤔

File: plugins.json [hunk 1]
```diff
@@ -1407,10 +1407,18 @@
     "githubUrl": "https://github.com/miguel-a-calles-mba/serverless-stack-termination-protection",
     "status": "none"
   },
-  {
+  { 
+    "name": "serverless-middleware",
+    "description": "Serverless plugin to allow middleware handlers configured directly in serverless.yaml",
+    "githubUrl": "https://github.com/juanjoDiaz/serverless-middleware"
```
[the remark this answers is not in the document]
Comment (EliuX) [reply plugins.json]: Hi @eahefnawy, it seems that it was removed before from the json, but it remained in the README. In this PR it was also removed from the README 👍

**Pull request merged**